navigation_and_routing/pubspec.yaml

```diff
@@ -5,19 +5,19 @@ version: 1.0.0+1
 environment:
   sdk: ">=2.12.0 <3.0.0"
 dependencies:
+  adaptive_navigation: ^0.0.3
```

Comment:
Should we add [`sort_pub_dependencies`](https://dart-lang.github.io/linter/lints/sort_pub_dependencies.html) to the lint stack to enforce this?

Reply:
I didn't know the policy of adding new lints